### PR TITLE
Create education-news.php

### PR DESCRIPTION
**Pull request by cdavis-ubc**

### CODE REVIEW

File: templates/education/education-news.php [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+			<div class="boxey-inner">
+				
+				<small class="modal-tags header-tags"><?php echo esc_html( $plain_tags_slug ); ?></small>
+
+				<a href="#modal_<?php echo absint( $post_id ); ?>" target="_blank" title="<?php the_title_attribute(); ?>" class="fancybox-inline"><img src="<?php echo esc_url( $post_thumbnail ); ?>" alt="" /></a>
+				
+				<h3 class="post-title">
+
+				<div class="iso-title"><a href="#modal_<?php echo absint( $post_id ); ?>" target="_blank" title="<?php the_title_attribute(); ?>" class="fancybox-inline"><?php the_title_attribute(); ?></a>
```
Comment (richardtape) [inline templates/education/education-news.php]: `the_title_attribute()` is to be used when you're outputting the title _attribute_. You've used it here as the content of the link, too. This is where, if you want the content of the link to be the title, you should use `the_title()` - i.e. `...title="<?php the_title_attribute(); ?>"...` is correct but `...class="fancybox-inline"><?php the_title_attribute(); ?></a>...` is not.

File: templates/education/education-news.php [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+				</div>
+
+				</h3>	
+
+				<div class="iso-description hidden"><?php echo wp_kses_post( get_the_excerpt() ); ?> 
+					<?php the_content(); ?></div>
+
+				<div class="boxey-excerpt"><?php echo wp_kses_post( get_the_excerpt() ); ?></div>
```
Comment (richardtape) [inline templates/education/education-news.php:73]: Same comment as before about formatting for the excerpt.

File: templates/education/education-news.php (hunk 1 above)
Comment (cdavis-ubc) [inline templates/education/education-news.php]: ok just updated it the the_title where needed

File: templates/education/education-news.php (hunk 2 above)
Comment (cdavis-ubc) [inline templates/education/education-news.php:73]: Plain text formatting is fine for the excerpt in this template. Thanks

**Comment by richardtape** [comment]
Merged and on Verf

**Pull request merged**